datus/cli/tutorial.py

```
@@ -183,36 +204,101 @@ def run(self):
-                    if err := result.get("error"):
+                    if error_message:
                         console.print(" ❌[bold]There are some errors in the processing:[/]")
-                        console.print(f"    {err}")
+                        console.print(f"    {escape(error_message)}")
                     else:
                         console.print(" ⚠️No metrics initialized")
             else:
                 console.print(" ❌[bold]Metrics initialization failed:[/]")
-                console.print(f"    {result.get('message')}")
+                console.print(f"    {escape(str(error_message))}")
             return True
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)